app/src/main/java/com/awscherb/cardkeeper/data/handler/BaseRealmHandler.java

```diff
@@ -4,67 +4,50 @@
 
 import java.util.List;
 
+import io.reactivex.Completable;
+import io.reactivex.Observable;
```

Comment:
Make sure you understand a difference between `Flowable` and `Observable` before making a choice.